### PR TITLE
Updated translations from Phrase

### PR DESCRIPTION
**Pull request by terraware-phrase**
Exported the most recent locale files from Phrase

### CODE REVIEW

File: src/strings/csv/es.csv [hunk 1]
```diff
@@ -618,6 +618,7 @@ ORG_NOTIFICATION_ACTION,"Para cambiar la preferencia, vaya a [""Ajustes > Organi
 ORGANIZATION,Organización
 ORGANIZATION_CREATE_FAILED,No se puede crear la organización.
 ORGANIZATION_CREATED_MSG,"Puede acceder a las organizaciones a las que pertenece haciendo clic en la flecha de la esquina superior derecha, junto a su perfil."
```
Comment (karthikbtf) [inline src/strings/csv/es.csv:620]: @sgrimm  - the `ORGANIZATION_CREATED_MSG` message was updated as well but not reflected here in the spanish changes. 
Not a new string, just an update.
Is there something we can do in the Phrase UI to trigger an update?

**Review by karthikbtf — APPROVED** [review]

File: src/strings/csv/es.csv (hunk 1 above)
Comment (sgrimm) [reply src/strings/csv/es.csv:620]: Ah, this was my mistake -- I hadn't set English as the "main language" for terraware-web in Phrase, so updates to English strings weren't causing the Spanish translations to be included in orders. Fixed now, and I'll manually place an order to cover the recently-modified strings.

Comment (karthikbtf) [reply src/strings/csv/es.csv:620]: Great, thanks Steve 🙏

Comment (sgrimm) [reply src/strings/csv/es.csv:620]: Done. The automator should place an order for these strings in the next hour or so.

**Pull request merged**